src/components/film-list-item/index.js

```diff
@@ -1,17 +1,34 @@
 import { Button } from '../index';
 import close from '../film-movie-form/movie-form/img/CloseButton.png';
 import { buttonContent } from '../../constants/const';
-import './film-list-item.css';
+import './style.css';
+import { useContext } from 'react';
+import { AppContext } from '../context';
+
+const FilmListItem = film => {
+  const { dispatch } = useContext(AppContext);
+  const { id, img, name, year, genres } = film;
+  const onDelete = () => {
+    dispatch({
+      type: 'delete',
+      payload: id
+    });
+  };
+
+  const handleInfo = () => {
+    dispatch({
+      type: 'info',
+      payload: film
+    });
+  };
+
+  const handleEdit = () => {
```

Comment:
The bracket can be removed.
```suggestion
  const handleEdit = () => 
```

Reply:
done